gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/purger/HivePurgerCommitPolicy.java

```diff
@@ -14,35 +14,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package gobblin.compliance;
+package gobblin.compliance.purger;
 
-import java.sql.SQLException;
-import java.util.List;
-
-import gobblin.util.HiveJdbcConnector;
+import com.google.common.base.Preconditions;
 
 
 /**
- * This class is responsible for executing Hive queries by initializing {@link HiveJdbcConnector}
+ * This commit policy checks if the lastModified time before and start of purging is same or not.
```

Comment:
Can you add something like " .... and only commits if the lastModified does not change.." along the line?